lib/mbta_v3_api/facilities/facility.ex

```diff
@@ -0,0 +1,53 @@
+defmodule MBTAV3API.Facilities.Facility do
+  @moduledoc "Data model and helpers corresponding to the MBTA V3 API Facility resource."
+  alias MBTAV3API.Stops.Stop
+  alias JsonApi.Item
+
+  defstruct id: nil,
+            type: nil,
+            short_name: nil,
+            long_name: nil,
+            stop: nil,
+            latitude: nil,
+            longitude: nil,
+            properties: []
+
+
+  @type id_t :: String.t()
```

Comment:
Is there a particular reason to alias `String.t()` here?

Reply:
Not really, mostly following a pattern established in the codebase.

Reply:
In my mind, there are two (admittedly relatively small) benefits.

The first is indirection, such that if you decide to change your ID to be an integer, you only need to change the type in one place in your codebase.

Also, it helps your type definitions be more semantically meaningful. If a function's type spec tells you that it takes `Facility.id_t()` as a parameter, you could try to pass in `Facility.short_name`, and maybe the type checker might say those both evaluate to strings and not complain (I don't actually know if it would be smarter than this or not), but as a human, you can see that's not what is expected.